src/app.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
 // Write your code here
+const readline = require('node:readline');
+const { generateRandomNumber } = require('../src/modules/generateRandomNumber');
+const {
+  checkIsValidUserInput,
+} = require('../src/modules/checkIsValidUserInput');
+const { getBullsAndCows } = require('../src/modules/getBullsAndCows');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const generatedNumber = generateRandomNumber();
+
+rl.question('Enter your number: ', (userInput) => {
+  if (checkIsValidUserInput(userInput)) {
+    // eslint-disable-next-line no-console
+    console.log(getBullsAndCows(userInput, generatedNumber));
```

Comment:
The `userInput` is currently passed as a string to `getBullsAndCows`, which expects numeric inputs. Consider converting `userInput` to a number before passing it to the function.